indico/modules/events/registration/controllers/display.py

```diff
@@ -409,7 +410,8 @@ def _check_access(self):
     def _process(self):
         self.registration.update_state(withdrawn=True)
         flash(_('Your registration has been withdrawn.'), 'success')
-        return redirect(url_for('.display_regform', self.registration.locator.registrant))
+        notify_registration_state_update(self.registration)
+        return redirect(url_for('event_registration.display_regform', self.registration.locator.registrant))
```

Comment:
Changing the redirect url here was necessary for the test to run. I was not able to get the test request context to descend into the event_registration blueprint. If you have ideas on how this can be done without changing the url please let me know!

Reply:
ah damn, I don't think there's a good solution that's not doing any awful hackery, so let's leave it as is...